tests/run-tests.sh

```diff
@@ -20,5 +20,18 @@ if [[ "${TEST_TYPE}" == "aio" ]]; then
                                                      -e ansible_connection=local \
                                                      ${TEST_DIR}/tests/playbooks/test.yml
 
-    source ${TEST_DIR}/scripts/deploy-all.sh
+    # Deploys specified toolset
+    case ${RE_JOB_ACTION} in
```

Comment:
I'm not sure how best to address this, but maybe we need to define `RE_JOB_ACTION` as a default so that if this script is executed and no environment variable is present, the script will run `deploy-all`, fail, etc?

Reply:
I've added a check, that will default to all if the environment variable is undefined.